sauce/features/accounts/toggle-splits/index.js

```diff
@@ -0,0 +1,85 @@
+import { Feature } from 'toolkit/core/feature';
+import * as toolkitHelper from 'toolkit/helpers/toolkit';
+
+function isNotSubTransaction(transaction) {
+  const displayItemType = transaction.get('displayItemType');
+  return displayItemType !== ynab.constants.TransactionDisplayItemType.SubTransaction &&
+          displayItemType !== ynab.constants.TransactionDisplayItemType.ScheduledSubTransaction;
+}
+
+export class ToggleSplits extends Feature {
+  get idName() {
+    return 'toggle-splits';
+  }
+
+  constructor() {
+    super();
+    this.expanded = false;
+  }
+
+  injectCSS() { return require('./index.css'); }
+
+  shouldInvoke() {
+    return toolkitHelper.getCurrentRouteName().indexOf('account') !== -1 && !$(`#${this.idName}`).length;
+  }
+
+  invoke() {
+    this.accountsController = toolkitHelper.controllerLookup('accounts');
+
+    const buttonText = toolkitHelper.i10n('toolkit.toggleSplits', 'Toggle Splits');
+    this.$button = $('<button>', { id: this.idName, class: 'ember-view button' })
+      .append($('<i>', { class: 'ember-view flaticon stroke right' }).toggle(!this.expanded))
+      .append($('<i>', { class: 'ember-view flaticon stroke down' }).toggle(this.expanded))
+      .append(buttonText)
+      .insertAfter('.accounts-toolbar .undo-redo-container');
+
+    $('.accounts-toolbar-left').addClass('toolkit-accounts-toolbar-left');
+
+    this.$button.click(() => this.toggle());
+
+    // Only want to wrap contentResults once
+    if (typeof this.accountsController.get('toolkitShowSubTransactions') === 'undefined') {
+      const ynabContentResults = this.accountsController.contentResults;
+      this.accountsController.reopen({
+        toolkitShowSubTransactions: this.expanded,
+        toolkitShowSubTransactionsChanged: Ember.observer('toolkitShowSubTransactions', function () {
+          Ember.run.debounce(() => {
+            this.notifyPropertyChange('contentResults');
+          }, 25);
+        }),
+        contentResults: Ember.computed(...ynabContentResults._dependentKeys, {
+          get: function () {
+            let contentResults = ynabContentResults._getter.apply(this);
+            if (!this.get('toolkitShowSubTransactions')) {
+              contentResults = contentResults.filter(isNotSubTransaction);
+            }
+            return contentResults;
+          },
+          set: function (_key, val) {
+            return val;
+          }
+        })
+      });
+    }
+
+    // Trigger default state
+    this.accountsController.notifyPropertyChange('contentResults');
+  }
+
+  onserve(changedNodes) {
```

Comment:
If we're currently testing the feature and it's working then we don't need an `observe` function as this one (`onserve`) won't be getting called right now.

Would be great to either rename it to `observe` if it's needed, or remove it if it's not.

Reply:
Good catch. I missed this in my testing. I'll check it out and see whether it causes any bugs not having it.